src/appkit.cs

```diff
@@ -12220,6 +12555,17 @@ public interface NSSegmentedControl {
 		[Mac (10,10,3)]
 		[Export ("doubleValueForSelectedSegment")]
 		double GetValueForSelectedSegment (); // actually returns double
+
+		[Mac (10,12)]
+		[Static]
+		[Export ("segmentedControlWithLabels:trackingMode:target:action:")]
+		NSSegmentedControl FromLabels (string[] labels, NSSegmentSwitchTracking trackingMode, [NullAllowed] NSObject target, [NullAllowed] Selector action);
```

Comment:
Maybe we could add some syntax sugar in here, we could turn `Selector action` into an actual `Action` same for the next one